gst/nnstreamer/ml_agent.c

```diff
@@ -87,7 +87,7 @@ mlagent_get_model_path_from (const GValue * val)
       g_autofree gchar *name = g_strdup (parts[MODEL_PART_IDX_NAME]);
       guint version = strtoul (parts[MODEL_PART_IDX_VERSION], NULL, 10);
       g_autofree gchar *stringfied_json = NULL;
-      g_autoptr (JsonParser) json_parser = NULL;
+      JsonParser *json_parser = NULL;
```

Comment:
Actually unnecessary change.
JsonParser defines auto release function by using g_object_unref (see below:)
https://github.com/GNOME/json-glib/blob/8410c3283ce3d79e6e54612b1b5761b05eedd9de/json-glib/json-parser.h#L194

Reply:
Thanks for your comment.
I checked G_DEFINE_AUTOPTR_CLEANUP_FUNC about JsonParser.

I'll close this PR.